gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,5 +1,5 @@
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-6.8.3-bin.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-8.3.0-bin.zip
```

Comment:
Missed this in the previous review - gradle always drops the patch version from the path if it is `.0`, so this should be:

```
distributionUrl=https\://services.gradle.org/distributions/gradle-8.3-bin.zip
```